gtsam/navigation/CombinedImuFactor.cpp

```diff
@@ -155,7 +156,9 @@ void PreintegratedCombinedMeasurements::integrateMeasurement(
   Matrix3 temp = vel_H_biasAcc * p().biasAccOmegaInt.block<3, 3>(3, 0)
       * theta_H_biasOmega.transpose();
   D_v_R(&G_measCov_Gt) = temp;
+  D_v_t(&G_measCov_Gt) = vel_H_biasAcc * (aCov / dt) * pos_H_biasAcc.transpose();
   D_R_v(&G_measCov_Gt) = temp.transpose();
+  D_t_v(&G_measCov_Gt) = pos_H_biasAcc * (aCov / dt) * vel_H_biasAcc.transpose();
```

Comment:
We missed these off-diagonal elements. I guess one test would be to do the full matrix multiplication and see the results, and then update accordingly.